framework/console/controllers/MigrateController.php

```diff
@@ -523,6 +523,35 @@ protected function parseFields()
         ];
     }
 
+    /**
+     * Splits field into chunks
+     *
+     * @param $field
+     * @return array|array[]|false|string[]
+     */
+    protected function splitFieldIntoChunks($field)
+    {
+        $hasDoubleQuotes = false;
+        $regex = "/defaultValue\(.*?:.*?\)/";
+        preg_match_all($regex, $field, $matches);
+        if (isset($matches[0][0])) {
```

Comment:
Can't `if` check be done for `preg_match_all` itself?

Reply:
I think it's safer to check `$matches[0][0]`.